templates/test/ci/cluster-template-prow-ci-version-dra.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     cni-windows: ${CLUSTER_NAME}-calico
     containerd-logger: enabled
     csi-proxy: enabled
-    metrics-server: enabled
```

Comment:
note to self: we should standardize so that the `windows: enabled` label is always present (I assume having this here ahead of time allows the addition of a Windows pool later in the cluster lifecycle) and that we remove this ClusterResourceSet metrics-server implementation on all our templates (I assume we're not using it anywhere in our test cases)